scripts/uploader.rb

```diff
@@ -8,9 +8,14 @@ class Uploader
     attr_reader :container, :directory, :type
 
     NIGHTLY_BUILD_RETENTION_TIME = 8.weeks
+    CONFIG_FILE = "#{__dir__}/../config/upload.yml"
```

Comment:
not necessarily for this PR, but should this be `File.expand_path(__dir__, "../config/upload.yml")` ? /cc @Fryguy

Reply:
Yes, that's preferable.  Alternately Pathname.new(__dir__).join(...)

Reply:
Updated to use Pathname